### PR TITLE
Updating README.md with version v1.1 and Test Coverage report

### PR DESCRIPTION
**Pull request by rgsubh**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,17 +1,67 @@
-name: "build-test"
-on: # rebuild any PRs and main branch changes
+on:
```
Comment (ds-ms) [inline .github/workflows/test.yml:1]: Have the test-coverage stuff in a different PR

Comment (ds-ms) [inline .github/workflows/test.yml:1]: whenever you're fixing tests raise for master first.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -1,17 +1,67 @@
...
+        $count = 0; 
+        foreach ($j in $coverage_result) 
+        { 
+          if ($j.tostring().startswith("--------------")) 
+          { 
+                if (!$start) 
+                { 
+                    $start = $true;
+                    $start_index = $count 
+                }
```
Comment (ds-ms) [inline .github/workflows/test.yml:46]: Did you author this script?

Comment (rgsubh) [reply .github/workflows/test.yml:46]: Nope: Reused from 
https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps.Docs?path=%2FPipelines%2FEnvironment%2FGitHub%20Actions%2FGitHub_Actions_CI.md&_a=preview

File: package.json [hunk 3]
```diff
@@ -5,7 +5,8 @@
     "main": "lib/run.js",
     "scripts": {
         "build": "tsc --outDir .\\lib\\ --rootDir .\\src\\",
-        "test": "jest"
+        "test": "jest",
+        "test-coverage": "jest --coverage"
```
Comment (ds-ms) [inline package.json:9]: Is there a way to supply a format or something here?

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -1,17 +1,67 @@
...
+                if (!$start) 
+                { 
+                    $start = $true;
+                    $start_index = $count 
+                } 
+                elseif (!$middle) 
+                {
+                    $middle = $true; 
+                    $middle_index = $count 
+                }
```
Comment (ds-ms) [inline .github/workflows/test.yml:51]: middle index is not being used anywhere right?

**Comment by ds-ms** [comment]
@rgsubh what do `%Stmts`, `%Branch`, `%Funcs` mean?

File: .github/workflows/test.yml (hunk 1 above)
Comment (rgsubh) [reply .github/workflows/test.yml:1]: Okay. will take care next time.

File: package.json (hunk 3 above)
Comment (rgsubh) [reply package.json:9]: I don't think jest gives that option.

**Pull request merged**